scripts/package-json-check.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 if [ ! -z "git diff --diff-filter=MRA --name-only | grep package.json" ]; then
   yarn lint-fix-package-json
-  echo "Changes made to package.json files. Please stage these changes before pushing."
+  git diff --diff-filter=MRA --name-only | grep package.json | xargs git add
```

Comment:
Is this related to the husky fix?

Reply:
Yes, previously this file only lint, but don't stage the change, which means, you have to edit the commit or make another commit for the change just made by lint